packages/components/tests/e2e/specs/vsCodeExtension.spec.js

```diff
@@ -841,26 +841,28 @@ context('VS Code Extension', () => {
       );
       cy.get('.trace-filter__arrows-text').contains('1 / 2 results');
 
-      cy.get('.trace-filter__input')
-        .focus()
-        .type('{downarrow}')
-        .type('{enter}');
+      cy.get('.trace-filter__input').type('{downarrow}').type('{enter}');
       cy.get('.trace-filter__input').should(
         'have.value',
         '"GET /admin/orders" "GET /admin" '
       );
       cy.get('.trace-filter__suggestions').should('not.be.visible');
       cy.get('.trace-filter__arrows-text').contains('1 / 3 results');
 
-      cy.get('.trace-filter__input').focus().type('{esc}');
+      cy.get('.trace-filter__input').type('{esc}');
       cy.get('.trace-filter__suggestions').should('not.be.visible');
       cy.get('.trace-filter__input').should('not.be.focused');
 
-      cy.get('.trace-filter__input').focus().type('select');
+      cy.get('.trace-filter__input').type('select');
       cy.get('.trace-filter__suggestions').should('be.visible');
       cy.get('.trace-filter__suggestions-item')
         .first()
         .should('contain.text', 'SELECT');
```

Comment:
This was previously passing even though a search for `SELECT` would fail. Can we have a test case which validates a search for `select` and `SELECT` return the same results?